RecoTracker/LSTCore/src/alpaka/MiniDoublet.h

```diff
@@ -1007,76 +1008,71 @@ namespace SDL {
       nTotalMDs = 0;
       alpaka::syncBlockThreads(acc);
 
-      // Initialize variables outside of the for loop.
-      int occupancy, category_number, eta_number;
+      // Occupancy matrix for 0.8 GeV pT Cut
+      constexpr int p08_occupancy_matrix[4][4] = {
+          {49, 42, 37, 41},      // category 0
+          {100, 100, 100, 100},  // category 1
```

Comment:
It seems like the last two occupancies in category 1 go unused, unless I made a mistake somewhere.

Reply:
I fixed this in my commit below. I haven't checked how much memory it saves though compared to the "master" branch.